examples/timeseries.md

```diff
@@ -3,13 +3,15 @@ title: Timeseries
 author: Tom Augspurger
 ---
 
+# Timeseries
+
 Pandas started out in the financial world, so naturally it has strong timeseries support.
 
 The first half of this post will look at pandas' capabilities for manipulating time series data.
 The second half will discuss modelling time series data with statsmodels.
 
 
-```{python}
+```python
```

Comment:
We decided to require braces in https://github.com/pystitch/stitch/pull/14 and https://github.com/pystitch/stitch/pull/19